src/SFA.DAS.Reservations.Api.Client/DependencyResolution/ReservationsClientApiConfigurationADAdapter.cs

```diff
@@ -21,5 +22,7 @@ public ReservationsClientApiConfigurationADAdapter(ReservationsClientApiConfigur
         public string ClientId => _config.ClientId;
         public string ClientSecret => _config.ClientSecret;
         public string IdentifierUri => _config.IdentifierUri;
+
+        public string ApiBaseUrl => _config.ApiBaseUrl;
```

Comment:
I assume this was introduced because it's set in the V2 version of the code and is required in IAzureActiveDirectoryClientConfiguration interface.

My only concern with this is the local dev code uses a stub of the reservations api via the configuration 'ReservationsClientApiConfiguration' (this stub is not using AAD) will this new upgrade allow us to continue using the stub reservations API. 

FYI but I'm sure you know, the reservations API client was written in this convoluted style because of the incompatibilities between the dot net standard System.Net.Http and the older framework version

Reply:
As per conversation offline, this looks to be ok